### ISSUE TITLE
Predictions were bad for the model trained on larger image resolution (2600 x 3300)

### ISSUE THREAD

**Issue by kalai2033** [pythonlessons/TensorFlow-2.x-YOLOv3#109]
Hi,
     Thanks for the tensorflow implementation of Yolo. I have used your code to train with my custom images of resolution 2600 x 3300 with two class objects for detection. I trained it for 100 epochs and tested the model. The results were very poor. The training graphs looks good i mean val loss are decreasing but the results were poor. Is it because of the image size since every image is being resized to 416 x 416 during training?

**Comment by pythonlessons** [comment]
Yes, you were right, you should increate training image size, try something 832x832 o even more

**Comment by kalai2033** [comment]
Thanks. May i know the maximum size that I can try training yolo?

**Comment by pythonlessons** [comment]
I think there is no limit, it depends on your hardware, larger the image more resources are required
